[editor]/move_cursor/move_cursor.lua

```diff
@@ -350,29 +328,27 @@ function getZoomSpeeds()
 end
 
 function disable()
-	if (not isEnabled) then
-		return false
+	if isEnabled then
+		removeEventHandler("onClientCursorMove", root, onClientCursorMove_cursor)
+		unbindControl("quick_rotate_increase", "down", rotateWithMouseWheel)
+		unbindControl("quick_rotate_decrease", "down", rotateWithMouseWheel)
+		unbindControl("zoom_in", "down", zoomWithMouseWheel)
+		unbindControl("zoom_out", "down", zoomWithMouseWheel)
+		isEnabled = false
 	end
-	-- remove events, unbind keys
-	removeEventHandler("onClientCursorMove", root, onClientCursorMove_cursor)
-	unbindControl("quick_rotate_increase", "down", rotateWithMouseWheel)
-	unbindControl("quick_rotate_decrease", "down", rotateWithMouseWheel)
-	unbindControl("zoom_in", "down", zoomWithMouseWheel)
-	unbindControl("zoom_out", "down", zoomWithMouseWheel)
-	isEnabled = false
+	return true
 end
 
 function enable()
-	if isEnabled then
-		return false
+	if not isEnabled then
+		ignoreFirst = true
+		addEventHandler("onClientCursorMove", root, onClientCursorMove_cursor)
+		bindControl("quick_rotate_increase", "down", rotateWithMouseWheel)
+		bindControl("quick_rotate_decrease", "down", rotateWithMouseWheel)
+		bindControl("zoom_in", "down", zoomWithMouseWheel)
+		bindControl("zoom_out", "down", zoomWithMouseWheel)
+		setTimer(processCursorMove, 50, 1)

```

Comment:
```suggestion
		setTimer(processCursorMove, 50, 1) --Lazy but we have to wait for MTA to switch modes
```
IMHO we must keep this comment, or someone could end up removing the timer in a future PR